presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java

```diff
@@ -23,13 +27,61 @@ public TestHiveWindowQueries()
         super(true);
     }
 
+    protected List<String> getQueries(String rankingFunction)
+    {
+        ImmutableList.Builder<String> queries = ImmutableList.builder();
+        List<String> columnProjections = Arrays.asList("orderkey, orderdate, totalprice");
+        List<String> overClauses = Arrays.asList("PARTITION BY orderkey ORDER BY totalprice",
+                "PARTITION BY custkey, orderkey ORDER BY totalprice",
+                "PARTITION BY orderdate ORDER BY orderkey asc, totalprice desc",
+                "PARTITION BY orderkey, orderdate ORDER BY totalprice asc nulls first",
+                "PARTITION BY orderkey, custkey ORDER BY orderdate asc nulls first, totalprice asc, shippriority desc",
+                "PARTITION BY custkey, orderkey, shippriority ORDER BY orderdate, totalprice asc nulls first",
+                "ORDER BY orderdate desc, totalprice asc, shippriority desc nulls first");
+
+        for (String columnProjection : columnProjections) {
+            for (String overClause : overClauses) {
+                queries.add(String.format("SELECT %s, %s OVER (%s) AS rnk FROM orders", columnProjection, rankingFunction, overClause));
+            }
+        }
+        return queries.build();
+    }
+
+    protected void testFunction(String functionName)
```

Comment:
Nit : Rename the functions to testRankingFunction and getRankingQueries as these functions could look very different from those for the value and aggregate window functions.

Reply:
Thanks for the suggestion, changed accordingly!